tests/db/test_bigchain_api.py

```diff
@@ -689,7 +689,7 @@ def test_transfer_single_owners_single_input_from_multiple_outputs(self, b, user
         assert len(tx_signed['transaction']['fulfillments']) == 1
         assert len(tx_signed['transaction']['conditions']) == 1
 
-    def test_single_current_owner_multiple_new_owners_single_input(self, b, user_sk, user_vk, inputs):
+    def test_single_current_owner_multiple_owners_after_single_input(self, b, user_sk, user_vk, inputs):
```

Comment:
The changed name still uses the "current_owner" phrase. Other test names also have the "current_owner" phrase.